app/src/main/java/com/example/trainingintive/repository/Schedulers.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.trainingintive.repository
+
+import com.example.trainingintive.di.IoScheduler
+import com.example.trainingintive.di.UiScheduler
+import io.reactivex.rxjava3.core.Scheduler
+import javax.inject.Inject
+
+class Schedulers @Inject constructor(
+    @IoScheduler private val ioScheduler: Scheduler,
```

Comment:
I suppose you are injecting schedulers here to be able to inject test schedulers in unit tests. You can achieve it in a simpler way, you can make this class open, and inherit from it with some TestSchedulers class. This way you can get rid of Schedulers Quailifiers. Then in unit test you will inject TestSchedulers class instead of Schedulers.